Formula/mongocli.rb

```diff
@@ -0,0 +1,31 @@
+class Mongocli < Formula
+  desc "MongoDB CLI enables you to manage your MongoDB in the Cloud"
+  homepage "https://github.com/mongodb/mongocli"
+  url "https://github.com/mongodb/mongocli/archive/refs/tags/v1.16.0.tar.gz"
+  sha256 "184f0a13ca0cfb642203727b4971622a761c77ac85066fc51689356690f5e2d0"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    with_env(
+      MCLI_VERSION: version.to_s,
+      MCLI_GIT_SHA: "homebrew-release",
+    ) do
+      system "make", "build"
+    end
+    bin.install "bin/mongocli"
+
+    (bash_completion/"mongocli").write Utils.safe_popen_read(bin/"mongocli", "completion", "bash")
+    (fish_completion/"mongocli.fish").write Utils.safe_popen_read(bin/"mongocli", "completion", "fish")
+    (zsh_completion/"_mongocli").write Utils.safe_popen_read(bin/"mongocli", "completion", "zsh")
+  end
+
+  test do
+    assert_match "mongocli version: #{version}", shell_output("#{bin}/mongocli --version")
+    assert_match "Atlas operations.", shell_output("#{bin}/mongocli atlas")
+    assert_match "Ops Manager operations.", shell_output("#{bin}/mongocli om")
+    assert_match "MongoDB Cloud Manager operations.", shell_output("#{bin}/mongocli cm")
```

Comment:
This is still just comparing help output, could we instead try to perform an operation but have it fail because there aren't any credentials, or something similar?

Reply:
added a failing command with missing credentials, also for context, `mongocli config ls` is an actual command that list your profiles, in this case empty so that was a safe command to run